### ISSUE TITLE
Typed event excludes space and punctuation

### ISSUE THREAD

**Issue by hughlang** [ryanisaacg/quicksilver#489]
**Describe the bug**

In event.rs, the ReceivedCharacter event only allows is_alphanumeric. This excludes space and punctuation. This criteria works better:

```rust
                glutin::WindowEvent::ReceivedCharacter(c) if !c.is_ascii_control() => {
                    events.push(Event::Typed(c));
                }
```
See: https://doc.rust-lang.org/std/primitive.char.html#method.is_ascii_control

**To Reproduce**

In the State run loop, try capturing the char inputs:
```rust
            Event::Typed(c) => {
                println!("Input char {}", c);
            }
```

**Environment and versions (please complete the following information):**
- Environment: macOS Mojave, Open GL 4.1
- Rust compiler version: 1.34
- Quicksilver verison: Latest

I assume all versions of OS, Rust, Quicksilver. I can submit a PR, but I'm in the middle of a massive feature build on my branch.

**Comment by ryanisaacg** [comment]
That makes sense! I wasn't aware of 'is_ascii_control'; I'll do this in a release in the near future.

**Comment by ryanisaacg** [comment]
This should be fixed in 0.3.12! Thanks for the suggestion.
